draft/authentication-oauth2.md

```diff
@@ -0,0 +1,256 @@
+---
+RFC: XXXX
+Author: Kyle Fuller
+Status: Draft
+Created: 2015-09-28
+Last Modified: 2015-09-29
+---
+
+# API Blueprint RFC XXXX: OAuth 2 Authentication Scheme
+
+## Table of Contents
+
+- [Abstract](#abstract)
+- [Motivation](#motivation)
+- [Rationale](#rationale)
+- [Backwards Compatibility](#backwards-compatibility)
+
+## Abstract
+
+This RFC adds the "OAuth 2" authentication scheme for the Authentication
+framework proposed in RFC XXXX. OAuth 2 is defined in
+[RFC 6749](http://tools.ietf.org/html/rfc6749).
+
+## Motivation
+
+OAuth 2 is a highly common authentication mechanism and it's oftenly used in APIs.
+
+## Rationale
+
+You can use the OAuth 2 Authentication scheme by defining it within the
+"Authentication Schemes" section of an API Blueprint.
```

Comment:
I consider `OAuth 2` to be similar to `Basic`. A user should be able to use it directly in the `Authenticated` sections with some default values.